portfolio/src/main/webapp/index.html

```diff
@@ -1,27 +1,183 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
-    <title>Hadley Shapland</title>
-    <link rel="stylesheet" href="style.css">
-    <!-- include jquery -->
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
-    <script src="script.js"></script>
-  </head>
-
-  <body>
-    <div id="sidebar"></div>
- 
-    <div id="content">
-      <h1>HADLEY SHAPLAND</h1>
-      <p>Welcome to my webpage! Check out my <a href="https://www.linkedin.com/in/hadley-shapland/">
-          LinkedIn</a> and my <a href="https://github.com/hadleyshapland">
-          GitHub</a> to learn more about me.
-      </p>
-      <h3>Contact</h3>
-      <p>hadley.a.shapland@vanderbilt.edu</p>
-      <p>217.418.9903</p>
-      
-    </div>
-  </body>
+<html lang="en">
+   <head>
+      <meta charset="utf-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+      <meta name="description" content="Personal Webpage" />
+      <meta name="author" content="Hadley Shapland" />
+      <title>Hadley Shapland</title>
+      <link rel="apple-touch-icon" sizes="180x180" href="assets/img/apple-touch-icon.png">
+      <link rel="icon" type="image/png" sizes="32x32" href="assets/img/favicon-32x32.png">
+      <link rel="icon" type="image/png" sizes="16x16" href="assets/img/favicon-16x16.png">
+      <link rel="manifest" href="assets/img/site.webmanifest">
+      <!-- Font Awesome icons (free version)-->
+      <script src="https://use.fontawesome.com/releases/v5.13.0/js/all.js" crossorigin="anonymous"></script>
+      <!-- Google fonts-->
+      <link href="https://fonts.googleapis.com/css?family=Saira+Extra+Condensed:500,700" rel="stylesheet" type="text/css" />
+      <link href="https://fonts.googleapis.com/css?family=Muli:400,400i,800,800i" rel="stylesheet" type="text/css" />
+      <!-- Core theme CSS (includes Bootstrap)-->
+      <link href="css/styles.css" rel="stylesheet" />
+   </head>
+   <body id="page-top">
+      <!-- Navigation-->
+      <nav class="navbar navbar-expand-lg navbar-dark bg-primary fixed-top" id="sideNav">
+         <a class="navbar-brand js-scroll-trigger" href="#page-top"
+            ><span class="d-block d-lg-none">Hadley Shapland</span><span class="d-none d-lg-block"><img class="img-fluid img-profile rounded-circle mx-auto mb-2" src="assets/img/profile.jpg" alt="" /></span></a
```

Comment:
nit: could you fill in the alt tag on the image?

Reply:
good point, done